### ISSUE TITLE
Documentation for logging should mention how to capture output in "capture output" section

### ISSUE THREAD

**Issue by charlesreid1** [ipython/ipython#11146]
In the [Session logging and restoring](http://ipython.readthedocs.io/en/stable/interactive/reference.html?highlight=logging#session-logging-and-restoring) section of the docs, there is a section called [Manual capture of command output and magic output](http://ipython.readthedocs.io/en/stable/interactive/reference.html?highlight=logging#manual-capture-of-command-output-and-magic-output). This does not mention how to capture iPython output. How to capture output seems important enough  to merit a mention. 

The logstart magic command will tell you you aren't logging output on startup, but there is no mention of how to enable it. The session logging says nothing about capturing iPython output. It is buried in the list of options to the logstart magic, [here](http://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-logstart), to which there is only one undecorated link in the middle of a paragraph.

**TL; DR:** Tell users they can capture iPython output by passing the`-o` flag to `%logstart`, instead of expecting them to visit an API reference. This is important information.

**Comment by takluyver** [comment]
Would you like to make a PR?

**Comment by charlesreid1** [comment]
I don't see what folder corresponds to the `Tutorials` section in the [docs](https://github.com/ipython/ipython/tree/master/docs/source) folder.

**Comment by charlesreid1** [comment]
it was in `interactive/reference.rst`.

#111147

**Comment by takluyver** [comment]
Closed by #11147 . Thanks!

If you need to change the docs again, the 'edit on Github' link at the top of the page should help you find the right file. This is a handy thing that readthedocs provides.
